docs/docs/use-cases/african-urbanization.md

```diff
@@ -0,0 +1,30 @@
+# African Urbanization
+
+_DRAFT: More info needed in sections below. This is just an outline for discussion._
+
+## Business Scenario
+
+**TODO**.
+
+## Technologies
+
+African Urbanization highlights usage of the followoing technologies and platforms:
+
+* **TODO**
+* **TODO**
+
+## Architecture
+
+**TODO**: Architecture diagram with some additional explainations go here.
+
+## Source Code
+
+The source code for the use case, including more detailed documentation, is available as open source on GitHub
+in the repositories:
+
+* [core-scaling-ui](https://github.com/qlik-oss/core-scaling-ui) - **TODO**.
+* [core-scaling](https://github.com/qlik-oss/core-scaling) - **TODO**.
+
+## Live Deployment
```

Comment:
I think it would be nice to have this at the very top, people tend to want to see the good stuff right away :)